### DISCUSSION TITLE
DwarFS Logo

### DISCUSSION THREAD

**Discussion by mhx** [mhx/dwarfs#179] [Ideas]
Following up on #151, it's about time that this project gets a proper logo.

Unfortunately, I suck at graphics. It seems the various image generating AIs suck far less than me, so I gave it a shot. As usual, when presented with a non-mainstream query, the results are pretty bad on average. However, one of the suggestions was actually quite reasonable:

![dwarfs-logo](https://github.com/mhx/dwarfs/assets/301215/d1cb5ef5-09b8-459a-a00f-a0eaa3d96f1d)

What do y'all think?

I'm happy to collect further logo suggestions in this thread.

Also, I'd be more than happy if someone would contribute a (non-AI generated) logo to the project.

Thanks!

**Comment by silentnoodlemaster** [comment]
I feel like the speed lines make it too complex at a small scale so I tried without, at the same time tried single color variants, white one works, black one looks off


![dwarfs2](https://github.com/mhx/dwarfs/assets/10667809/90ca5f9a-946c-4440-b504-4f8259d7e554)
![dwarfs_w](https://github.com/mhx/dwarfs/assets/10667809/98e62b4c-fb47-4d81-8736-c8c404f333ef)
![dwarfs_b](https://github.com/mhx/dwarfs/assets/10667809/97e689a6-3d2d-4926-88e4-862ce7d6c48f)

**Comment by Phantop** [comment]
I have very... mixed feelings on using AI art (albeit, I understand why it makes sense in lieu of anyone with the skills to actually make any). I do like the concept, though, of the dwarf holding storage platters. If it were me, I would sooner not use a logo than use AI art, but this definitely is an appealing option.

**Reply by Phantop** [reply]
Looking at the issue, in terms of icons for view in file managers, I definitely think sticking to common 'image-file' icons would be best on that front, too.
